tests/unit/util/newconfig/ArrayViewTests.cpp

```diff
@@ -78,6 +78,10 @@ TEST_F(ArrayViewTest, IterateArray)
     EXPECT_EQ((*it++).asString(), "125.5.5.2");
     EXPECT_EQ((*it++).asString(), "204.2.2.2");
     EXPECT_EQ((it), arr.end<ValueView>());
+
+    auto itArray = configData.getArray("array.[].sub");
```

Comment:
Better to make it as a separate test and also check that they are not always equal.